.env.template

```diff
@@ -0,0 +1,13 @@
+# To use this environment variables file, make a copy of this file named ".env" then add your environment values
```

Comment:
rename to "env.template" so that this file will appear on mac. (Comment from a previous CRDC-DH-BACKEND PR)

Reply:
The latest commit has renamed ".env.template" to "env.template"